dist/index.html

```diff
@@ -4,34 +4,33 @@
     <link rel="stylesheet' href='src/css/base.scss">
     <link href="https://fonts.googleapis.com/css2?family=Creepster&family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css2?family=Abril+Fatface&display=swap" rel="stylesheet">
-    <!-- <link rel="stylesheet" href="node_modules/@glidejs/glide/dist/css/glide.core.min.css"> -->
-    <!-- <script src="node_modules/@glidejs/glide/dist/glide.min.js"></script> -->
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Voyager</title>
   </head>
   <body>
     <main>
-      <section class="login-page hidden" id=loginPage>
+      <section class="login-page" id=loginPage>
         <section class="login-photo-left-side">
           <section class="logo-title">
-            <img class="application-logo" src="./images/travel-icon.svg" alt="Globe icon with location pins"/>
+            <img class="application-logo" src="./images/travel-icon.svg" alt="Globe icon with location pins" />
             <h1 class="application-title">Voyager</h1>
           </section>
         </section>
         <section class="login-right-side">
           <h1>Sign In</h1>
           <h2>Sign in to manage your adventures</h2>
-          <input class="user-name-input" id="userNameInput" type="text" placeholder="username" required/>
-          <input class="password-input" id="passwordInput" type="text" placeholder="password" required/>
+          <input class="user-name-input" id="userNameInput" type="text" placeholder="username" required />
+          <input class="password-input" id="passwordInput" type="password" placeholder="password" required />
+          <h2 class="login-feedback" id="loginFeedback"></h2>
```

Comment:
Curious what this feedback is. Does it get populated with a "login successful" or an error message type thing?